docs/details/commands/pmm-agent.md

```diff
@@ -46,6 +46,7 @@ Most options can be set via environment variables (shown in parentheses).
 | `--machine-id=machine-id`              | `PMM_AGENT_SETUP_MACHINE_ID`        | Node machine ID (default is auto-detected).
 | `--metrics-mode=auto`                  | `PMM_AGENT_SETUP_METRICS_MODE`      | Metrics flow mode for agents node-exporter. Can be `push` (agent will push metrics), `pull` (server scrapes metrics from agent) or `auto` (chosen by server).
 | `--node-model=NODE-MODEL`              | `PMM_AGENT_SETUP_NODE_MODEL`        | Node model.
+| `--paths-base=PATH`                    | `PMM_AGENT_PATHS_BASE`              | Base path for PMM client.
```

Comment:
could you please be more explicit here and define what it is, that is is actually path to all binaries ? Because ppl could read it as BASE for all other PATHs that would be relative

Reply:
Added. @PaulJacobs-percona please can you check description of flag here: https://github.com/percona/pmm-doc/pull/573/files#diff-ff428c96162b07ed27ea02832cbea69ae4aa4dbedd861426bd8a82c9535d24f2R68 and make it more clear for customers? Thanks.

Reply:
Basically if you pass absolute path to any param then it will be respected, othervise it will be based on paths-base param. Expect case when you pass exporter base and paths base together. Paths base will be used every where, but for exporters will be respected exporters base value.